src/components/Filters/Summary.js

```diff
@@ -131,6 +131,7 @@ class FilterSummary extends React.Component {
             return (
               <Box mb={10} mr={10} key={rule.id}>
                 <FilterDescription
+                  schema={this.props.schema[rule.name]}
```

Comment:
This should also be added to the ViewsMenu.js and be passed to the FilterDescription component (so that we can save an restore views).

Reply:
Good spot. We need TS here :(